### DISCUSSION TITLE
New OS window with different configuration

### DISCUSSION THREAD

**Discussion by kakawait** [kovidgoyal/kitty#5034] [Q&A] [answered]
I'm using _Kitty_ on MacOs and thanks to trick using _hammerspoon_ I was able to get _Kitty_ as _Quake_ / _Tilda_ mode (idea and concept can be found here https://github.com/kovidgoyal/kitty/issues/45)

To get best visual result I'm using `-o hide_window_decorations=yes`

<img width="1512" alt="Screenshot 2022-04-29 at 17 36 13" src="https://user-images.githubusercontent.com/275609/165977299-8fc86ae9-eb9f-4e80-954f-e51f92cda078.png">

However, I'd like when spawning a new OS window manually using task bar or `⌘ + n` (nothing related to [`init.lua`](https://gist.github.com/kakawait/7051d8b841c26c2d65b499a0872e380d) new window), to do not inheritate this preference?

You can find my [`init.lua`](https://gist.github.com/kakawait/7051d8b841c26c2d65b499a0872e380d) if want to see how I spawn the _Kitty_ process but is simply

```
/Applications/kitty.app/Contents/MacOS/kitty -1 -d $HOME -o hide_window_decorations=yes ...
```

I understand is a weird problem, I'm looking for any kind of solution or workaround even if I've to rethink my initial spawning process

**Comment by kovidgoyal** [comment]
hide_window_decorations is process global. If you want new kitty windows with decorations, start a new kitty process.

**Reply by kakawait** [reply]
Maybe what can I do is to unmap `cmd+n` (since I never used task bar with mouse) and remap to spawn a new process?

Off-topic question, is possible to override `map` configuration using _cli_ and `-o`?

**Comment by kovidgoyal** [comment, marked as answer]
Sure see https://sw.kovidgoyal.net/kitty/launch/
